setup.cfg

```diff
@@ -7,6 +7,7 @@ test = pytest
 addopts =
     --doctest-modules
     --disable-pytest-warnings
+    -r s
```

Comment:
Could we please use the long form options?

Reply:
I changed 's' to 'skipped' (that apparently works), but for "-r" I don't see any long format in the pytest help